docs/UserGuide.md

```diff
@@ -109,11 +109,28 @@ In the example above, <span class="main-command">add</span> is the command word
 | INDEX        |  None  | The index of the endpoint as shown in the endpoint panel list         |
 | THEME        |  None  | The theme for the application                                         |
 | METHOD       |   -x   | The [request method](#84-request-methods) to use for an endpoint      |
-| ADDRESS      |   -u   | The URL to use for an endpoint                                        |
+| URL          |   -u   | The URL to use for an endpoint                                        |
 | HEADER       |   -h   | The header to use for an endpoint **(must be enclosed with \"\")**      |
 | DATA         |   -d   | The data to use for an endpoint **(must be in [JSON](#85-json-format) format)**          |
 | TAG          |   -t   | The tag to label an endpoint                 |
 
+Here are some general rules to follow when entering prefixes and parameters:
+* A whitespace must be included before every prefix.<br>
+  e.g. `-x METHOD-u URL` is not acceptable, and ` -x METHOD -u URL` is in the correct format.<br>
+  
+* Parameters can be in any order.<br>
+  e.g. If the command specifies `-x METHOD -u URL`, `-u URL -x METHOD` is also acceptable.<br>
+  
+* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+  e.g. If you specify `-u https://github.com/ -u https://google.com/`, only `-u https://google.com/` will be taken.<br>
+  
+* To add multiple parameters of the same prefix, add the prefix multiple times before each parameter.<br>
+  e.g. To add two TAGs, enter `-t tagOne -t tagTwo`.<br>
+  e.g. To add three HEADERSs, enter `-h "header:one" -h "header:two" -h "header:three"`.<br>
```

Comment:
Double quotations need to be escaped with `\` so that it shows up correctly on the jekyll page. The whitespace between colon and value should also be added to be consistent with the header format.